### PR TITLE
Burn close all

### LINKED ISSUE #651: delete "close all" button
don't need anymore with new design (request from Design)

### PR DESCRIPTION
**Pull request by sastels**
resolves #651 

Think I got all of the related code too.

These changes didn't affect the tests 😞

### CODE REVIEW

**Comment by maxneuvians** [comment]
https://github.com/cds-snc/vac-benefits-directory/blob/71825429602ca1f252bcef25c945b159bfbf7ff9/components/favourites.js#L207

File: components/BB.js [hunk 1]
```diff
@@ -298,21 +283,9 @@ export class BB extends Component {
                     </FormControl>
                   </Grid>
 
-                  <Grid item xs={8} className={classnames(classes.collapse)}>
```
Comment (smcmurtry) [inline components/BB.js]: I think there's a corresponding grid item that should have `xs={4}` changed to `xs={12}`

**Review by maxneuvians — APPROVED** [review]
LGTM 🦑

**Comment by sastels** [comment]
Urg, for some reason the cards don’t have any space between them in this PR. Will fix this before merging...
Ah, it’s because benefitlist is just a bunch of Cards, and I deleted the grid container that they were in.

**Pull request merged**